### ISSUE TITLE
Why not use airspeed sp for gain scaling if airspeed disabled?

### ISSUE THREAD

**Issue by CarlOlsson** [PX4/Firmware#11661]
Currently we are scaling the actuators with the trim airspeed (a constant value) if airspeed is disabled in the fixed wing attitude controller
https://github.com/PX4/Firmware/blob/b54ca67de9b5900c7ed11bef323c7b4e5a0339b6/src/modules/fw_att_control/FixedwingAttitudeControl.cpp#L480-L481

It seems sub optimal since the demanded airspeed can be significantly higher if the fixed wing position controller requests it due to [ground speed undershoot in high winds](https://github.com/PX4/Firmware/blob/b54ca67de9b5900c7ed11bef323c7b4e5a0339b6/src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp#L496-L498). Is there a reason the scaling is not done using the demanded airspeed instead?

**Comment by LorenzMeier** [comment]
No, I think that's just historical. I think a test with demanded airspeed would be worthwhile.

**Comment by Antiheavy** [comment]
Sorry for my ignorance, but how can airspeed demand be reasonably estimated if airspeed is disabled in the controller?

I asked a related question in this other thread: https://github.com/PX4/Firmware/issues/11677

**Comment by CarlOlsson** [comment]
> how can airspeed demand be reasonably estimated if airspeed is disabled in the controller?

For some vehicles a reasonable estimate of airspeed can be acquired from the throttle setting scaled with air density and voltage

**Comment by Antiheavy** [comment]
Still might be interesting to look into this.  FYI @Kjkinney
